.github/workflows/check.yml

```diff
@@ -125,20 +125,21 @@ jobs:
         submodules: true
     - name: Create logs dir
       run: mkdir test-logs
-    - name: autogen.sh
-      run: NOCONFIGURE=1 ./autogen.sh
     - name: configure
       # We don't do gtk-doc or GObject-Introspection here, because they can
       # clash with AddressSanitizer. Instead, the clang build enables those.
       run: |
-        mkdir _build
-        pushd _build
-        ../configure  --enable-internal-checks --enable-asan --disable-introspection --without-curl
-        popd
+        meson _build \
+          -Db_sanitize=address,undefined \
+          -Dgir=disabled \
+          -Dgtkdoc=disabled \
+          -Dhttp_backend=soup \
+          -Dinternal_checks=true \
+          ${NULL+}
```

Comment:
Are the `${NULL+}`'s really necessary?

Reply:
It's a way to make all of the lines with content be the same (trailing backslash) so that adding or removing  a line doesn't cause diffstat on all the other lines. I'd prefer to keep that property.

(This is not new, we already do this for the build pass that already uses Meson.)

`${NULL+}` instead of `${NULL}` is a habit I've got into because I often write shell scripts under `set -eu`, where `${NULL}` would usually cause an error because that variable isn't defined (or if it is defined, its value will not be what we want). `${NULL+}` always expands to nothing, regardless of whether `NULL` is unset (likely) or empty (unlikely) or non-empty (pathological).

Reply:
Sure, it's a bit nicer IMO.